library/alpine

```diff
@@ -0,0 +1,9 @@
+# maintainer: Glider Labs <team@gliderlabs.com> (@gliderlabs)
+
+edge: git://github.com/gliderlabs/docker-alpine@8f429d1283f6897a1c21d5745dca8fa25d394a8b versions/edge
+
+3.1: git://github.com/gliderlabs/docker-alpine@e119cc4675afe3abe221b8a23144888c02601f4b versions/v3.1
```

Comment:
Didn't you want to add a `latest` alias for `3.1` here?  (functionally, this is accomplished by duplicating the line and changing the `3.1:` at the start to be `latest:`)